app/lib/screens/study/onboarding/study_selection.dart

```diff
@@ -143,6 +159,10 @@ class _StudySelectionScreenState extends State<StudySelectionScreen> {
                             child: StudyTile.fromStudy(
                               study: study,
                               onTap: () async {
+                                if (study.isClosed) {
+                                  await showStudyClosedDialog(context);
+                                  return;
+                                }
                                 await navigateToStudyOverview(context, study);
```

Comment:
are closed studies shown in study selection? should we make them invisible instead?